tools/generator/test/Fixtures.swift

```diff
@@ -2144,19 +2164,24 @@ $(INTERNAL_DIR)/targets/a1b2c/A 2/A.link.params
                     "@executable_path/../Frameworks",
                 ],
                 "MACOSX_DEPLOYMENT_TARGET": "11.0",
-                "OTHER_CFLAGS": [
-                    "-ivfsoverlay",
-                    "$(OBJROOT)/bazel-out-overlay.yaml",
-                ],
-                "OTHER_CPLUSPLUSFLAGS": [
-                    "-ivfsoverlay",
-                    "$(OBJROOT)/bazel-out-overlay.yaml",
-                ],
                 "OTHER_LDFLAGS": "@$(DERIVED_FILE_DIR)/link.params",
+                "OTHER_SWIFT_FLAGS": #"""
+-vfsoverlay $(OBJROOT)/bazel-out-overlay.yaml
+"""#,
+                "PREVIEWS_SWIFT_INCLUDE_PATHS__": """
+$(PREVIEWS_SWIFT_INCLUDE_PATHS__NO)
+""",
+                "PREVIEWS_SWIFT_INCLUDE_PATHS__NO": "$(BUILD_DIR)/bazel-out/x",
+                "PREVIEWS_SWIFT_INCLUDE_PATHS__YES": """
+$(BUILD_DIR)/bazel-out/x $(PREVIEWS_SWIFT_INCLUDE_PATHS__NO)
+""",
```

Comment:
This is setting the same path twice. I should make sure the path isn't already in the base ones before doing this.